libpod/container_internal_linux.go

```diff
@@ -49,11 +49,9 @@ func (c *Container) mountSHM(shmOptions string) error {
 }
 
 func (c *Container) unmountSHM(mount string) error {
-	if err := unix.Unmount(mount, unix.MNT_DETACH); err != nil {
+	if err := unix.Unmount(mount, 0); err != nil {
```

Comment:
I got this already in #3931 - I think the implementation there is a bit safer because it properly reports errors

Reply:
Removed